ddl/multi_schema_change_test.go

```diff
@@ -1298,7 +1298,7 @@ func (c *cancelOnceHook) OnJobUpdated(job *model.Job) {
 	}
 	c.triggered = true
 	errs, err := ddl.CancelJobs(c.s, []int64{job.ID})
-	if errs[0] != nil {
+	if errs != nil && errs[0] != nil {
```

Comment:
```suggestion
	if len(errs) > 0 && errs[0] != nil {
```

Checking the length is less prone to error.

Reply:
It is an array which length is always bigger than 0.

Reply:
I mean if you check the length, such a restriction can be relax when you do the refactoring in the future.

Reply:
good point.